sdk/keyvault/azure-security-keyvault-secrets/src/private/keyvault_secrets_common_request.hpp

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Provides a wrapper class for the Azure Core Pipeline for all Key Vault services where
```

Comment:
For all services? Is this comment accurate?

Isn't this file only accessible and specific to keyvault-secrets? How will other service SDKs use this?